tests/test_expectation_maximization.py

```diff
@@ -0,0 +1,173 @@
+"""Test bayesian expectation maximization process."""
```

Comment:
Test names seem good, and testing for shape a good start. They can be added to as the functions are developed, including quite involved tests that check they do the things we think they do with specifically crafted input.